lib/tasks/verify-readme.js

```diff
@@ -7,13 +7,20 @@ const {
 const path = require("path");
 const isCI = require("is-ci");
 const vfile = require("vfile");
+const fs = require('fs-extra');
 const remark = require("remark");
 const remarkLint = require("remark-lint");
 const origamiPreset = require("remark-preset-lint-origami-component");
 const report = require("vfile-reporter");
 
 async function origamiJson(config) {
+	// Error if there is no readme to verify.
 	const readmePath = path.join(config.cwd, "/README.md");
+	const hasReadme = await fs.exists(readmePath);
+	if (!hasReadme) {
```

Comment:
an error was previously thrown because of the below `readFile`, but this is more friendly